projects/igniteui-angular/src/lib/grids/grid/grid-row.component.html

```diff
@@ -127,10 +127,12 @@
         class="igx-grid__td igx-grid__td--fw igx-grid__td--tree-cell"
         [class.igx-grid__td--edited]="rowID | transactionState:col.field:grid.rowEditable:grid.transactions:grid.pipeTrigger:grid.crudService.cell"
         [class.igx-grid__td--number]="col.dataType === 'number' && col.visibleIndex !== 0"
+        [class.igx-grid__td--new]="addRow && !rowData[col.field]"
```

Comment:
check and use the new hostbinding introduced by @MayaKirova with the tree grid PR